src/widgets/main/solution/section/style.module.scss

```
@@ -2,84 +2,143 @@
 
 .solution {
 	@include token.flex_column;
-	@include token.flex_center;
-	width: 100%;
-	gap: 28px;
-	color: token.$color_text_inverse;
+	align-items: center;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)